playwright/test/view-items.test.ts

```diff
@@ -1,15 +1,27 @@
-import { multiVolumeItem } from './contexts';
+import { multiVolumeItem, itemWithSearchAndStructures } from './contexts';
 import { isMobile } from './actions/common';
 import { volumesNavigationLabel } from './text/aria-labels';
 import {
   zoomInButton,
+  rotateButton,
   openseadragonCanvas,
   fullscreenButton,
+  searchWithinResultsHeader,
+  mainViewer,
+  accordionItemLicense,
+  accordionItemVolumes,
+  accordionItemContents,
 } from './selectors/item';
 import { baseUrl } from './helpers/urls';
 
 const domain = new URL(baseUrl).host;
 
+const searchWithinForm = '#searchWithin';
```

Comment:
what are the deciding factors for when to put selectors into their own file? Just when they get complex enough?

Reply:
I still don't think we should be selecting by form. Is there any benefit to doing so? If there are things labelled the same on the page, that should be at least a chance to discuss why from a UX POV?